src/content/en/fundamentals/primers/service-workers/a-basic-service-worker.markdown

```diff
@@ -0,0 +1,183 @@
+  how a client gets, or registers, a service worker. It shows you how a service 
+  worker prepares to act as a proxy for its clients. Both subjects have more 
+  depth than is shown in this section. It's not intended to be comprehensive; 
+  It's intended to give you a basic foundation on which to build service worker 
+  knowledge.</p>
+
+{% include shared/toc.liquid %}
+
+{% include shared/takeaway.liquid list=page.key-takeaways.tldr %}
+
+## A client registers a service worker
+
+To use a service worker, a client tells the browser to install it by calling 
+`register()`. Every page that needs the service worker must implement register(), 
+but only the first page to do so will trigger the download. The most basic 
+register implementation looks like this:
+
+{% highlight javascript %}
+//Does the browser support service workers?
+if ('serviceWorker' in navigator) {
+  //Yes
+  navigator.serviceWorker.register('/service-worker.js').then(function() {
+    // Advanced actions we'll get to later.
+  });
+} else {
+  //No, but this client should work anyway.
+}
+{% endhighlight %}
+
+Generally you'll put this code at the end of a page's <body> element so that it 
+doesn't block page rendering. Do this with either an inline script or a link to 
+a separate JavaScript file.
+
+There's a little more that needs to be said about clients. Now let's talk about 
+the service worker itself.
+
+{% include shared/remember.liquid title="Aside" list=page.notes.promises %}
+
+## Where can the service worker play?
+
+A client tells the service worker where it can operate. This is called the 
+service worker's scope, and it's the lowest level of your website where the the 
+service worker can be used. Let's look at the registration code again, but with 
+some changes to the register() parameters.
+
+{% highlight javascript %}
+//Does the browser support service workers?
+if ('serviceWorker' in navigator) {
+  //Yes
+  navigator.serviceWorker.register('service-worker.js', {scope: '/'})
+    .then(function() {
+    // Advanced actions we'll get to later.
+  });
+} else {
+  //No, but this client should work anyway.
+}
+{% endhighlight %}
+
+This code is functionally identical to the example given in the last section. 
+Both examples specify that the scope of the service worker is the entire domain. 
+If I want to restrict the service worker to part of a site, I simply specify it 
+in the scope. 
+
+For example, say that I had an auction site with different features for buyers 
+and sellers. The buyers section is at `example.com/buyers` and the sellers section 
+is at `example.com/sellers`. A service worker with a scope of `/buyers/` it can only 
+serve clients under example.com/buyers. It cannot serve clients under 
+`example.com/sellers`. Similarly, a service worker with a scope of `/sellers/` can 
+only serve clients under `example.com/sellers`.
+
+{% include shared/remember.liquid title="Aside" list=page.notes.https-only %}
```

Comment:
'Aside' is an idiom I was trying to introduce. I actually tried to create aside.liquid, but I couldn't get it to work.